_includes/portfolio_grid.html

```diff
@@ -1,6 +1,6 @@
 <!-- Portfolio Grid -->
 {% if site.locale and site.locale != "" and site.locale != nil %}
-<section class="bg-light page-section" id="{{ site.data.sitetext[site.locale].portfolio.section | default: "portfolio" }}">
+<section class="bg-light page-section" id="{{ site.data.sitetext.portfolio.section | default: "portfolio" }}">
```

Comment:
This change looks incorrect, can you explain why `[site.locale]` was removed if this wasn't a typo?

Reply:
## Summary
The issue is that the styling is attached to the id value but styling is only added to the English id value in SASS.

## Full Explanation
The line I changed sets the "id" value from the _data.  This id is used in the CSS to style the portfolio grid section "portfolio". [(check /_sass/layout/_portfolio.scss line 2 and line 66)](https://github.com/raviriley/agency-jekyll-theme/blob/master/_sass/layout/_portfolio.scss).

Since english has the value `portfolio`. That work. Other languages have different values for `portfolio.section`. This means the `#portfolio` CSS styles will not be applied in those languages.

The change I made ensures the id is always set to "portfolio".

An alternative solution would to remove the id "#portfolio" in the SASS code on the lines I mentioned above. With this change, you can revert removing `[site.locale]`.